### ISSUE TITLE
MSFT-201[Registration] JAWS : There is only one region

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#45]
**Steps to Reproduce**
1. Launch JAWS 
2. Launch https://accounts.topcoder.com/member/registration in Windows / FF 
3. Navigate through “R”

**Expected Result**
But there are more than one regions in the page.
Banner
Main region / Form region
Content information (Have an account and Privacy policy area)
Hope there should be another region where the “Github, Google and Facebook” links are presented

**Actual Result**
Actual announcement: 
There is only one region called “Banner”

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Firefox
**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FJAWS%20only%20one%20region
